### PR TITLE
deps: update gapic generator java to 2.40.0

### PR DESCRIPTION
**Pull request by JoeWang1127**

### CODE REVIEW

**Review by blakeli0 — APPROVED** [review]

File: java-admanager/ad-manager-bom/pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>com.google.api-ads</groupId>
+        <groupId>com.google.api-ads.api.grpc</groupId>
```
Comment (blakeli0) [inline java-admanager/ad-manager-bom/pom.xml:27]: Is this intentional?

File: java-admanager/pom.xml [hunk 2]
```diff
@@ -46,4 +46,4 @@
     <module>ad-manager-bom</module>
   </modules>
 
-</project>
+  </project>
```
Comment (blakeli0) [inline java-admanager/pom.xml:49]: Can we revert this change as well? Also do we know where this change is from?

Comment (JoeWang1127) [reply java-admanager/pom.xml:49]: This also comes from `fix-poms.py`.

pom.xml is not rendered from templates when updates so it's not clear to me how this indentation come.

**Review by blakeli0 — APPROVED** [review]
